viewing_party/party.py

```diff
@@ -1,23 +1,171 @@
 # ------------- WAVE 1 --------------------
+"""Create and modify movie data"""
+
 
 def create_movie(title, genre, rating):
-    pass
+    """create a movie dictionary"""
+    if title is None or genre is None or rating is None:
+        return None
+    return {"title": title, "genre": genre, "rating": rating}
+
+
+def add_to_watched(user_data, movie):
+    """append movie dictionary to user_data "watched" list"""
+    user_data["watched"].append(
+        create_movie(movie["title"], movie["genre"], movie["rating"])
+    )
+    return user_data
+
+
+def add_to_watchlist(user_data, movie):
+    """append movie dictionairy to user_data "watchlist" list"""

```

Comment:
>dictionairy

There are some extensions for spell-checking in code. Here's the one I use: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker

Reply:
haha, whoops! Installed it, thanks!